vcpkg.json

```diff
@@ -5,6 +5,16 @@
   "homepage": "https://mixxx.org/",
   "license": "GPL-2.0-or-later",
   "supports": "!uwp",
+  "vcpkg-configuration": {
+    "overlay-ports": [
+      "overlay/ports",
+      "overlay/osx",
+      "overlay/windows"
```

Comment:
One note on the implementation: This exposes all overlay ports on all platforms. These ports (and corresponding dependencies) should, however, already be scoped by platform, therefore this should be a non-issue.